jest.snapshot-resolver.js

```diff
@@ -0,0 +1,22 @@
+// @ts-check
+
+const path = require('path')
+
+const SNAPSHOT_EXTENSION = '.snap'
+const TEST_EXTENSION = '.tsx'
```

Comment:
Do we need to make a switch here based on the environment we're in? bazel vs outside bazel

```suggestion
const TEST_EXTENSION = isBazel ? '.js' : '.tsx'
```

Reply:
It looks like this won't be possible like I originally thought because jest needs the mapping going both ways. I thought we only need to convert `*.[jt]sx?` to a snapshot filename but it looks like jest requires that we also convert the snapshot name back to the original. When given a `.js` we don't know the original file extension 😞